plugins/src/main/java/com/google/firebase/gradle/plugins/SemVerTask.kt

```diff
@@ -33,8 +32,6 @@ abstract class SemVerTask : DefaultTask() {
   @get:Input abstract val currentVersionString: Property<String>
   @get:Input abstract val previousVersionString: Property<String>
 
-  @get:OutputFile abstract val outputApiFile: RegularFileProperty
```

Comment:
Would you mind adding a TODO here to add cache functionality later (no need to do it in this PR).

Ideally, the output should be the result of running metalava, but that might need a bit of finessing.

Reply:
Sure thing